packages/toolkit/src/view/knowledge/components/tabs/ChunkTab.tsx

```diff
@@ -50,23 +55,25 @@ export const ChunkTab = ({ knowledgeBase, onGoToUpload }: ChunkTabProps) => {
     accessToken,
   });
 
-  const { data: allFiles, isLoading: isLoadingFiles } =
-    useListKnowledgeBaseFiles({
-      namespaceId: selectedNamespace,
-      knowledgeBaseId: knowledgeBase.catalogId,
-      accessToken,
-      enabled: enabledQuery && Boolean(me.data?.id),
-    });
-
-  const updateChunkMutation = useUpdateChunk();
+  const {
+    data: filesData,
+    isLoading,
+    refetch,
+  } = useListKnowledgeBaseFiles({
+    namespaceId: selectedNamespace,
+    knowledgeBaseId: knowledgeBase.catalogId,
+    accessToken,
+    enabled: enabledQuery && Boolean(me.data?.id),
+    pageSize: 100,
+  });
 
   const files = React.useMemo(() => {
-    return (
-      allFiles?.filter(
-        (file) => file.processStatus !== "FILE_PROCESS_STATUS_FAILED",
-      ) || []
+    return (filesData?.files || []).filter(
+      (file) => file.processStatus !== "FILE_PROCESS_STATUS_FAILED",
     );
-  }, [allFiles]);
+  }, [filesData]);
```

Comment:
Could you update this yo use `isSuccess` from `useListKnowledgeBaseFiles`?
And then we should use `[knowledgeBaseFiles.isSuccess, knowledgeBaseFiles.data]` as deps.